htdocs/web_portal/controllers/utils.php

```diff
@@ -382,10 +382,12 @@ function getSGroupDataFromWeb() {
  * @return array an array representation of a service
  */
 function getSeDataFromWeb() {
-    // Fields that are used to link other objects to the site
+    
     $fields = array (
             'serviceType',
-            'endpointUrl'
+            'IS_MONITORED',
+            'NOTIFY',
+            'PRODUCTION_LEVEL'
```

Comment:
Why are these now in `$fields`?

Reply:
Because the only thing preventing them being there before was the need to treat their validation differently as they were booleans. This isn't an issue any more so we can simplify things.

Reply:
Does this pull request add the new boolean validation? Seems like thats add in 82deb49ea2b01265cc972fb2f957a13d84b0fda3 in #161...

Reply:
True. This pull request depends on #161 and #161 depends on this request. Sorry.